scripts/build-plugin-list.js

```diff
@@ -71,7 +71,6 @@ function extractPlugins(pluginContent) {
     const name = match[1]
     const url = match[2]
     const description = match[3] ? match[3].trim().replace(/ {2,}/g, ' ') : ''
-
```

Comment:
Can you add back the deleted line, please

Reply:
Done